src/components/Dashboard/CompleteFiling.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div class="complete-ia-filing body-2">
+    <h4>Incorporation Complete</h4>
+
+    <p>{{name}} has been successfully incorporated.</p>
+    <p>Return to your Manage Businesses dashboard to access your business and file changes.</p>
+
+    <div class="to-dashboard-container">
+      <v-btn
+        color="primary"
+        @click.native.stop="returnToDashboard()"
+        >
+        <span>Return to Dashboard</span>
+      </v-btn>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { mapState } from 'vuex'
+import { Component, Vue, Prop } from 'vue-property-decorator'
+import { ErrorContact } from '@/components/common'
```

Comment:
Obsolete?

Reply:
Good catch! and I thought I managed to clean this up. Will update.